core/scripts/local/DepositCollateral.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @notice Deposits collateral into existing token position with --collateral of collateral.
+ *
+ * Example: `$(npm bin)/truffle exec ./scripts/local/DepositCollateral.js --network test --collateral 25 --emp 0x6E2F1B57AF5C6237B7512b4DdC1FFDE2Fb7F90B9`
+ */
+const { toWei, toBN } = web3.utils;
+const { MAX_UINT_VAL } = require("../../../common/Constants");
+
+// Deployed contract ABI's and addresses we need to fetch.
+const ExpiringMultiParty = artifacts.require("ExpiringMultiParty");
+const ExpandedERC20 = artifacts.require("ExpandedERC20");
+const argv = require("minimist")(process.argv.slice(), { string: ["emp", "collateral", "collateralToken"] });
+
+async function depositCollateral(callback) {
+  try {
+    if (!argv.emp || !argv.collateral) {
+      throw new Error(`
+      required: --emp must be the emp address.
+      required: --collateral must be the amount of collateral to supply to back the tokens.
+      optional: --collateralToken must be the address of the deployed collateral currency
```

Comment:
nit: same as above -- is this being used?

Reply:
Nope